### PR TITLE
Little cleaning of some scripts

### PR DESCRIPTION
**Pull request by gagou9**

### CODE REVIEW

File: scripts/db.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 print_usage_and_exit() {
   echo "Usage: $0 [dump|import|drop_and_import] filename [args_for_mysql]"
-  echo "For example: $0 --user=USER --password=PASS --host=HOST"
+  echo "For example: $0 import mysql-dump-2015-03-15.sql --user=USER --password=PASS --host=HOST"
```
Comment (jfly) [inline scripts/db.sh]: This isn't a great filename, as this script is meant to deal with `*.tar.gz` files.

File: scripts/db.sh [hunk 2]
```diff
@@ -12,7 +12,8 @@ fi
 COMMAND=$1
 shift
 if [ "$COMMAND" != "dump" ] && [ "$COMMAND" != "import" ] && [ "$COMMAND" != "drop_and_import" ]; then
-  echo "Unrecognized command: $COMMAND. Must be one of dump, import, drop_and_import."
+  echo "Unrecognized command: $COMMAND."
+  print_usage_and_exit
   exit 1
```
Comment (jfly) [inline scripts/db.sh]: Since we just called `print_usage_and_exit`, there's no need for this `exit 1` anymore.

**Comment by jfly** [comment]
Thanks Gael! I made a couple of little comments.

**Comment by gagou9** [comment]
just applied your comments :)

**Comment by jfly** [comment]
Can you resolve conflicts with cubing/master? In https://github.com/cubing/worldcubeassociation.org/commit/b93de33933d1a6d5179950cf08f620ab78747d6a, I changed the database export to deal with folders instead of `.tar.gz` files, so I think that change is conflicting with yours.

**Comment by gagou9** [comment]
okayyy that was hard. I had a hard time understanding how to go back to cubing/worldcubeassociation.org on my local repo to then be able to add my changes... All that work for such small changes !! :)

I hope it will do this time!

**Comment by jfly** [comment]
Ahh, git can be difficult sometimes. This looks great, thanks!

**Pull request merged**